.flake8

```diff
@@ -0,0 +1,12 @@
+[flake8]
+max-line-length = 120
+
+# D100  Missing docstring in public module
+# D101  Missing docstring in public class
+# D102  Missing docstring in public method
+# D103  Missing docstring in public function
+# D104  Missing docstring in public package
+# D105  Missing docstring in magic method
+# D107  Missing docstring in __init__
+ignore = D100,D101,D102,D103,D105,D107
```

Comment:
Let’s not do doc linting at all on plugins. If it’s so many lines of code that it needs documenting there’s something fundamentally wrong.

Reply:
Don't know how to do it other than that.

Reply:
Me neither. Whatever.